### PR TITLE
add custom icons for switchImplIntf

### LINKED ISSUE #379: "Switch implementation/interface" button takes up obnoxiously much space in the tab bar
Every other extension that I have, which puts buttons there, uses icons, which don't take up too much space. Currently, without an icon the "Switch implementation/interface" takes up the space of 5 such icons and two tabs, which is extremely inconvenient.

### PR DESCRIPTION
**Pull request by ulugbekna**
fixes #379

### CODE REVIEW

**Comment by ulugbekna** [comment]
image

image
monokai:
image

File: assets/switch-impl-intf.dark.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  <rect fill="none" id="canvas_background" height="18" width="18" y="-1" x="-1"/>
+  <g display="none" overflow="visible" y="0" x="0" height="100%" width="100%" id="canvasGrid">
+   <rect fill="url(#gridpattern)" stroke-width="0" y="0" x="0" height="100%" width="100%"/>
+  </g>
+ </g>
+ <g>
+  <title>Layer 1</title>
+  <text xml:space="preserve" text-anchor="start" font-family="Helvetica, Arial, sans-serif" font-size="8" id="svg_1" y="6.15" x="-0.15624" stroke-width="0" stroke="#000" fill="#ffffff">ml</text>
+  <text xml:space="preserve" text-anchor="start" font-family="Helvetica, Arial, sans-serif" font-size="8" id="svg_2" y="15.71249" x="5.90625" stroke-width="0" stroke="#000" fill="#ffffff">mli</text>
+  <line stroke="#ffffff" stroke-linecap="undefined" stroke-linejoin="undefined" id="svg_4" y2="15.39999" x2="0.78126" y1="1.08751" x1="15.15624" fill="none"/>
```
Comment (mnxn) [inline assets/switch-impl-intf.dark.svg:14]: It looks a little bright compared to the other icons right now. If you use `#c5c5c5`, it'll be the same color as other icons:

image

```suggestion
  <text xml:space="preserve" text-anchor="start" font-family="Helvetica, Arial, sans-serif" font-size="8" id="svg_1" y="6.15" x="-0.15624" stroke-width="0" stroke="#000" fill="#c5c5c5">ml</text>
  <text xml:space="preserve" text-anchor="start" font-family="Helvetica, Arial, sans-serif" font-size="8" id="svg_2" y="15.71249" x="5.90625" stroke-width="0" stroke="#000" fill="#c5c5c5">mli</text>
  <line stroke="#c5c5c5" stroke-linecap="undefined" stroke-linejoin="undefined" id="svg_4" y2="15.39999" x2="0.78126" y1="1.08751" x1="15.15624" fill="none"/>
```

**Review by mnxn — CHANGES_REQUESTED** [review]
Needs a change log entry.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## 1.2.1
```
Comment (mnxn) [inline CHANGELOG.md:3]: Lets call this "Unreleased" for now.

```suggestion
## Unreleased
```

File: assets/switch-impl-intf.light.svg [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  </g>
+ </g>
+ <g>
+  <title>Layer 1</title>
+  <text fill="#424242" stroke="#000" stroke-width="0" x="-0.15624" y="6.15" id="svg_1" font-size="8" font-family="Helvetica, Arial, sans-serif" text-anchor="start" xml:space="preserve">ml</text>
+  <text fill="#424242" stroke="#000" stroke-width="0" x="5.90625" y="15.71249" id="svg_2" font-size="8" font-family="Helvetica, Arial, sans-serif" text-anchor="start" xml:space="preserve">mli</text>
+  <line fill="none" stroke="#424242" x1="15.15624" y1="1.08751" x2="0.78126" y2="15.39999" id="svg_4" stroke-linejoin="undefined" stroke-linecap="undefined"/>
+  <line fill="none" stroke="#000" x1="27.78123" y1="9.275" x2="27.84373" y2="9.275" id="svg_5" stroke-linejoin="undefined" stroke-linecap="undefined"/>
+ </g>
+</svg>
```
Comment (mnxn) [inline assets/switch-impl-intf.light.svg:18]: Github shows me this file is missing a "newline at end of file", but the other icon isn't.

**Review by mnxn — CHANGES_REQUESTED** [review]

File: assets/switch-impl-intf.light.svg (hunk 3 above)
Comment (ulugbekna) [reply assets/switch-impl-intf.light.svg:18]: 🤷‍♂️

**Review by mnxn — APPROVED** [review]

**Review by smorimoto — APPROVED** [review]

**Comment by smorimoto** [comment]
Good stuff!

**Pull request merged**